### PR TITLE
Fix english developing docs translation

### PR DESCRIPTION
**Pull request by Wojberni**
Hi, I have fixed some issues with developing documentation for English version. There was a bug for accessing the developing documentation. I fixed this and also fixed the bug when accessing the Q&A sidebar. Now it doesn't disappear after accessing the Q&A section. I did some minor visual improvements, like adding dots and commas, translating text from Chinese to English and formatting text using the markdown notation.
I built the project on my computer using node version 16.13.1 and everything was working fine. I hope that my contribution will be merged soon.
Best Regards,
Wojciech

### CODE REVIEW

**Review by flycash — APPROVED** [review]

**Comment by flycash** [comment]
conflict=.=

**Comment by flycash** [comment]
can you help to delete the yarn.lock file ? I think we don't need to use yarn.

**Comment by Wojberni** [comment]
Yes, I can delete it, but I am not sure, if the project will run with that old .lock file. It was updated while generating the documentation locally and in other framework like ruby on rails you should commit those files.

**Comment by flycash** [comment]
ohhh, I already deleted it. 😄

**Comment by Wojberni** [comment]
I deleted it too :laughing: So are we ready to merge? :rocket:

**Comment by flycash** [comment]
it looks like you delete wrong files. I mean only delete yarn.lock and I have already deleted it, you need to do nothing.
But you can not delete other files in the v2.1.0 directory and cache files. you should check the change files. 🥲

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ module.exports = {
     "/": {
       lang: "en-US",
       title: "Beego",
-      description: "The most easy use framework",
+      description: "The easiest to use framework",
```
Comment (flycash) [inline docs/.vuepress/config.js:21]: Do not change this file except this fix.

**Comment by flycash** [comment]
you can git pull origin main and then fix the conficts by access my change(the origin main)

File: docs/.vuepress/config.js (hunk 1 above)
Comment (Wojberni) [reply docs/.vuepress/config.js:21]: You meant, not to change the config.js file?

**Comment by flycash** [comment]
may be I can help to fix it. let me create a merge request to your branch

**Comment by Wojberni** [comment]
I will rebase onto your changes and fix it locally

**Comment by flycash** [comment]
https://github.com/Wojberni/beego-doc/pull/1

**Comment by flycash** [comment]
Thank you !

**Comment by Wojberni** [comment]
I rebased, it looks fine, if there are any issues with my changes please comment on them

**Comment by Wojberni** [comment]
You can delete the tmp branch and when the changes will be pulled onto the website?

**Pull request merged**